int32 ind
float32 xt
float32 yt
bool update
---
int32 count
float32[] x
float32[] y
int32[] priority